### PR TITLE
7.13.0 release notes

### PR DESCRIPTION
**Pull request by pugnascotia**
This PR adds the release notes for the 7.13.0 release.

~Note that we still need a release highlight for GeoIP v2.~

### Previews
* Release highlights: https://elasticsearch_72306.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/7.13/release-highlights.html
* Migration guide: https://elasticsearch_72306.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/7.13/migrating-7.13.html
* Release notes: https://elasticsearch_72306.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/7.13/release-notes-7.13.0.html

### CODE REVIEW

**Review by jrodewig — APPROVED** [review]
Thanks @pugnascotia. I pushed a commit with a few edits. The commit also contains release highlights for the following changes:

* Speed up terms aggregations when alone: https://github.com/elastic/elasticsearch/pull/69377
* Introduce `combined_fields` query: https://github.com/elastic/elasticsearch/pull/71213
* GeoIP V2

If those changes look okay to you, I'm good.

I'm also going to add @elastic/ml-docs as reviewers so they can take a look at the ML RNs.

File: docs/reference/release-notes/highlights.asciidoc [hunk 1]
```diff
@@ -37,4 +37,90 @@ Other versions:
...
+
+* It has no parent or child aggregations.
+* Target indices have no deleted documents or were recently merged.
+* You don't use document-level security.
+* There's no top-level query.
+* The field has global ordinals, such as a `keyword` or `ip` field.
+* There are fewer than a thousand distinct terms.
+
+Many time series use cases are likely to meet these criteria, particularly for
+data in the cold or frozen tier.
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights.asciidoc:124]: FYI @nik9000. Relates to https://github.com/elastic/elasticsearch/pull/69377

File: docs/reference/release-notes/highlights.asciidoc [hunk 2]
```diff
@@ -37,4 +37,90 @@ Other versions:
...
+[discrete]
+[[new-combined-fields-query-type]]
+=== New `combined_fields` query type
+
+7.13 introduces the `combined_fields` query, new a Query DSL query type for
+searching multiple `text` fields as a combined field. You can use the
+`combined_fields` query as a simpler alternative to the `multi_match` query's
+`cross_fields` type option. See the
+{ref}/query-dsl-combined-fields-query.html[`combined_fields` query
+documentation].
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights.asciidoc:108]: FYI @jtibshirani. Relates to https://github.com/elastic/elasticsearch/pull/71213.

File: docs/reference/release-notes/highlights.asciidoc [hunk 3]
```diff
@@ -37,4 +37,90 @@ Other versions:
...
+[discrete]
+[[automatic-updates-geoip-processor]]
+=== Automatic updates for the GeoIP processor
+
+By default, {es} now automatically downloads GeoIP2 database updates for the
+`geoip` processor from the Elastic GeoIP endpoint:
+https://geoip.elastic.co/v1/database. You can use the new
+{ref}/geoip-stats-api.html[GeoIP stats API] to get download stats for these
+updates. For more information, see the updated
+{ref}/geoip-processor.html[`geoip` processor documentation].
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights.asciidoc:97]: FYI @probakowski @martijnvg

File: docs/reference/release-notes/highlights.asciidoc [hunk 4]
```diff
@@ -37,4 +37,90 @@ Other versions:
...
+`combined_fields` query as a simpler alternative to the `multi_match` query's
+`cross_fields` type option. See the
+{ref}/query-dsl-combined-fields-query.html[`combined_fields` query
+documentation].
+
+[discrete]
+[[faster-terms-aggregations]]
+=== Faster `terms` aggregations
+
+We've improved the speed of the `terms` aggregation by up to 92% when:
```
Comment (nik9000) [inline docs/reference/release-notes/highlights.asciidoc:114]: "up to" isn't right - there isn't really a max. If all these hold true then we return the answer in nearly constant time. Its not *quite* constant. If you have more segments and we have to sum up segment metadata. But it's pretty darn low. You can basically make the index as big as you like and it'll still take 30ms to answer.

Comment (jrodewig) [reply docs/reference/release-notes/highlights.asciidoc]: Thanks for catching this @nik9000. I've updated this to just state:

`We've significantly improved the speed of the `terms` aggregation when: ...`

**Comment by nik9000** [comment]
Lgtm

**Comment by mark-vieira** [comment]
Thanks for the updates @jrodewig!

File: docs/reference/release-notes/highlights.asciidoc (hunk 3 above)
Comment (pugnascotia) [reply docs/reference/release-notes/highlights.asciidoc:97]: @jakelandis does this section relate to the geo feature that shouldn't be highlighted in 7.13?

**Review by szabosteve — COMMENTED** [review]
ML parts LGTM, thanks!

File: docs/reference/release-notes/highlights.asciidoc (hunk 3 above)
Comment (jakelandis) [reply docs/reference/release-notes/highlights.asciidoc:97]: @pugnascotia - Actually, it needs to be pulled all together from the 7.13 release notes. We found a last minute issue and will delay this feature by at least 1 release.

Comment (pugnascotia) [reply docs/reference/release-notes/highlights.asciidoc:97]: I raised PR #72743

**Comment by jrodewig** [comment]
Backported the RNs and migration guide changes to 7.x with #72763. That PR also backports the removal of GeoIPv2 RNs with #72743.

**Pull request merged**